fsharp-backend/src/BackendOnlyStdLib/LibDarkInternal.fs

```diff
@@ -93,37 +99,35 @@ let fns : List<BuiltInFn> =
               return result |> List.map DStr |> DList
             }
           | _ -> incorrectArgs ())
-      sqlSpec = NotYetImplementedTODO
+      sqlSpec = NotQueryable
       previewable = Impure
       deprecated = NotDeprecated }
 
-
+    // WHATISTHIS
     { name = fn "DarkInternal" "checkAllCanvases" 0
```

Comment:
Can be removed after migration, it wasn't a good way to do this.

Reply:
all the "oldinternal" functions are simmilar, no longer useful and will be removed later (can't remove them now without updating tests to account for it, so easier to leave them in for now)